appveyor.yml

```diff
@@ -28,3 +28,19 @@ after_build:
 
 build_script:
 - cmd: build-vs2017.cmd
+
+artifacts:
+- path: build\*.nupkg
+  name: core
+
+deploy:
+- provider: NuGet
+  api_key:
+    secure: Please insert key here
+  skip_symbols: false
+  artifact: core 
+  on:
+      appveyor_repo_tag: true
```

Comment:
@jonorossi - you are right. 

There is this ugly ass way of doing it. It would involve querying the version number from the [GitHub Api](https://www.appveyor.com/docs/build-worker-api/#update-build-details). I don't like this because we would have to write a client which involves exposing secure keys. 

Are you happy to conventionalise `APPVEYOR_REPO_TAG_NAME` to be the semver version?